### PR TITLE
[LTG-9743] update docs for webhook v2

### PR DESCRIPTION
**Pull request by michalbric**

### CODE REVIEW

File: delivery-api-v1.0.raml [hunk 1]
```diff
@@ -205,15 +229,73 @@ types:
...
+      - STOP_DELIVERED
+      - STOP_UNDELIVERED
+      - RIDE_PROCESSING
+      - RIDE_ACCEPTED
+      - RIDE_WAITING
+      - RIDE_ON_BOARD
+      - RIDE_COMPLETED
+      - RIDE_CANCELLED
+      - RIDE_REJECTED
+    description: Action correspond to changes in possible ride or stop states
```
Comment (nikodem-peter) [inline delivery-api-v1.0.raml]: grammar: actions or corresponds

File: delivery-api-v1.0.raml [hunk 2]
```diff
@@ -128,6 +128,9 @@ types:
       stops:
         type: Stop[]
         description: must start an element with the kind "pickup", followed by 1+ elements with the kind "destination" and followed by 0+ "fallback_destinations".
+      freezeDestinationsOrder?:
+        type: boolean
+        description: Delivery order of stops is fixed. (do not optimize it)
```
Comment (nikodem-peter) [inline delivery-api-v1.0.raml]: I would add default value, please.

Comment (michalbric) [reply delivery-api-v1.0.raml]: I should actually remove this. I copied the current state of the private docs but this is not related to the feature and I am not sure it is released yet.

**Review by nikodem-peter — APPROVED** [review]

File: delivery-api-v1.0.raml (hunk 2 above)
Comment (nikodem-peter) [reply delivery-api-v1.0.raml]: makes sense.

**Pull request merged**